extensions/amp-access/0.1/iframe-api/README.md

```diff
@@ -87,14 +116,37 @@ The `config` argument in the `connect` method will contain the original document
 }
 ```
 
+When used with `amp-subscriptions` the `pageProtocol` is set
```

Comment:
Hmm. I don't remember "pageProtocol" in the data structure. Are you sure? Isn't the protocol provided as an `init()` argument?

Reply:
oops you are right I was stashing it in pageProtocol in my example, will update.